atomisticparsers/gromacs/parser.py

```diff
@@ -640,6 +673,42 @@ def __init__(self):
         ]
         super().__init__()
 
+    def get_pbc(self):
+        pbc = self.input_parameters.get("pbc", "xyz")
+        return ["x" in pbc, "y" in pbc, "z" in pbc]
+
+    def get_mdp_file(self):
+        """
+        Tries to find the mdp input parameters (ext = mdp) that match the mainfile calculation.
+        Priority is as follows:
+            1. output mdp file containing both the matching mainfile name and the standard
+            gromacs name `mdout`
+            2. file containing the standard gromacs name `mdout`
+            3. input mdp file matching the mainfile name (as usual)
+            4. any `.mdp` file within the directory (as usual)
+        """
+        ext = "mdp"
```

Comment:
perhaps put these in init? easier to maintain

Reply:
done.